src/main/webapp/app/entities/text-exercise/text-exercise-detail.component.html

```diff
@@ -28,16 +28,16 @@ <h2><span jhiTranslate="arTeMiSApp.textExercise.detail.title">Text Exercise</spa
             </div>
         </dd>
         <dt><span jhiTranslate="arTeMiSApp.exercise.problemStatement">Problem Statement</span></dt>
-        <dd>
-            <span>{{ textExercise.problemStatement }}</span>
+        <dd class="editor-outline-background">
```

Comment:
Why not set the inner HTML of the `dl`?
```suggestion
        <dd class="editor-outline-background" [innerHTML]="formattedProblemStatement">
```
This can also be done in lines 39-40.

Reply:
sure - adjusted the changes